### PR TITLE
[UWP] Label MaxLines doesn't work

### LINKED ISSUE #4458: [UWP] Label MaxLines doesn't work
### Description
Label MaxLines should display 4 lines and then truncation, but it only shows one line. IF I remove the `LineBreakMode="TailTruncation"` then I get 4 lines.

### Steps to Reproduce
Run this and use the 3.3 page.

https://github.com/davidortinau/TheLittleThingsPlayground/blob/master/TheLittleThingsPlayground/Views/ThreeThreePage.xaml#L37

```
<Label x:Name="TruncatedLabel" MaxLines="4" LineBreakMode="TailTruncation" Text="You should see 4 lines of text and truncation at the end. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam mattis quam non enim pellentesque, ut placerat purus finibus. Nulla quis tincidunt ante. Ut mauris lectus, aliquam a sagittis vitae, consequat eget elit. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque convallis nunc nisi, a imperdiet elit efficitur et. Duis in lectus mollis, interdum ipsum et, tincidunt orci. Fusce ipsum metus, imperdiet non lacus vitae, facilisis feugiat magna. Nulla volutpat nisl tortor, a consectetur felis consectetur non. Curabitur in enim vulputate sem volutpat bibendum id nec lorem. Mauris laoreet lacus ac volutpat tempus."/>
```

### Expected Behavior
I should see 4 lines ending with truncation "...".

### Actual Behavior
I see 1 line ending with truncation "...".

### Basic Information

- Version with issue: 3.4.0.1008975

### Screenshots

image

After removing `LineBreakMode="TailTruncation"`:

image

### Reproduction Link

https://github.com/davidortinau/TheLittleThingsPlayground/blob/master/TheLittleThingsPlayground/Views/ThreeThreePage.xaml#L37

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change ###

In `UpdateLineBreakMode` take into account that `MaxLines` can be set and we do need to allow wrapping then truncating.

### Issues Resolved ### 

- fixes #4458

### API Changes ###

 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

`MaxLines` together with `LineBreakMode` now work on UWP!

### Before/After Screenshots ### 

screenshot 2018-11-23 22 56 28

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: Xamarin.Forms.Platform.UAP/LabelRenderer.cs [hunk 1]
```diff
@@ -282,22 +282,30 @@ void UpdateLineBreakMode(TextBlock textBlock)
...
 					textBlock.TextTrimming = TextTrimming.WordEllipsis;
-					textBlock.TextWrapping = TextWrapping.NoWrap;
+					DetermineTextWrapping(textBlock);
 					break;
 				default:
 					throw new ArgumentOutOfRangeException();
 			}
 		}
 
+		void DetermineTextWrapping(TextBlock textBlock)
```
Comment (paymicro) [inline Xamarin.Forms.Platform.UAP/LabelRenderer.cs]:
```suggestion
		void DetermineTruncatedTextWrapping(TextBlock textBlock)
```

Comment (paymicro) [reply Xamarin.Forms.Platform.UAP/LabelRenderer.cs]: It is necessary to change the names of this method in calls...

Comment (jfversluis) [reply Xamarin.Forms.Platform.UAP/LabelRenderer.cs]: Haha yeah. I thought that to myself right after accepting the suggested change... I'll do it now

**Comment by rmarinho** [comment]
build

**Comment by velocitysystems** [comment]
@samhouts Can this possibly make it into a 3.4.0 SR or 3.5.0?

**Comment by samhouts** [comment]
@velocitysystems You should see this in 3.5.0! Thanks!

**Pull request merged**